defaults.yml

```diff
@@ -110,7 +113,7 @@ drupal:
       # Database connection for Drupal.
       database:
         # This value is required, but a default is set in the-build.xml.
-        # database: drupal
+        database: db
```

Comment:
Do we need to change the default in the-build.xml instead of overriding it by default here?

Reply:
Instead of setting a default value here for the database name (which could break multisite behavior), I updated the default for the first database that gets copied with `defaults/install/the-build/build.yml`. See https://github.com/palantirnet/the-build/pull/218/commits/90b7d7db6829c0a127fd9cdada31b9f0de9ed1a9

Now, when a multisite is added, and that default is not present, it defaults to the site directory name (as it did before this PR).